### ISSUE TITLE
Update for Solr 5

### ISSUE THREAD

**Issue by DaniG2k** [sunspot/sunspot#658]
I am wondering if there will be an update to this great gem for Solr 5 at some point? :smile: would be great

**Comment by davekapp** [comment]
I'd love to see that as well. Unfortunately I haven't seen much activity on this library in a while. I don't know if it's actively maintained at this point or not.

**Comment by mois3x** [comment]
+1 for this. Solr 5 has good improvements, is more smart and simple. Even it is a standalone server by default now.

**Comment by backus** [comment]
I have sunspot working with Solr 5.  I'll clean things up a bit and open a pull request soon.

**Comment by mois3x** [comment]
Thanks, John. That definitely would be great.

**Comment by backus** [comment]
Opened the PR

**Comment by DaniG2k** [comment]
Great work, thanks :)

**Comment by mois3x** [comment]
Nice.

**Comment by victorsosa** [comment]
+1 for this

**Comment by IslamAzab** [comment]
:+1:

**Comment by Dragsbaek** [comment]
Any updates on this?

**Comment by IslamAzab** [comment]
^ @njakobsen

**Comment by Dragsbaek** [comment]
<div style="max-width: 500px;" id="_giphy_KsHExnwJ6I2cM"></div><script>var _giphy = _giphy || []; _giphy.push({id: "KsHExnwJ6I2cM",w: 236, h: 157});var g = document.createElement("script"); g.type = "text/javascript"; g.async = true;g.src = ("https:" == document.location.protocol ? "https://" : "http://") + "giphy.com/static/js/widgets/embed.js";var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(g, s);</script>

**Comment by backus** [comment]
I think you can close this now. #675 was merged into master

**Comment by brevilo** [comment]
Well, not until someone upgraded Jetty to v9 as requested by this issue :grin:
